numpy/core/defchararray.py

```diff
@@ -29,7 +29,7 @@
 import numpy
 
 __all__ = [
-    'chararray', 'equal', 'not_equal', 'greater_equal', 'less_equal',
+    'equal', 'not_equal', 'greater_equal', 'less_equal',
```

Comment:
`chararray` is added to the namespace in [core.__init__.py](https://github.com/numpy/numpy/blob/v1.16.1/numpy/core/__init__.py#L108) so this is a duplicate and confuses the autosummary